packages/shr-cli/app.js

```diff
@@ -190,35 +179,10 @@ if (doDD) {
 }
 
 let fhirResults = null;
-if (doES6 || doFHIR){
+if (doFHIR){
```

Comment:
Now that we have two separate conditionals with the same condition, could you put this statement inside the conditional at line 187?

Reply:
Oh yes. I didn't even notice that. Good catch. I'll make that change.